Cargo.toml

```diff
@@ -138,12 +154,11 @@ jemallocator = "0.5.4"
 
 [dev-dependencies]
 assert_fs = "1.0.13"
-chrono = { version = "0.4.31", features = ["serde"] }
+chrono = "0.4.38"
 env_logger = "0.10.1"
-jsonwebtoken = { version = "8.3.0-surreal.1", package = "surrealdb-jsonwebtoken" }
-opentelemetry-proto = { version = "0.2.0", features = [
+jsonwebtoken = "9.3.0"
```

Comment:
How does this work with the WASM build? We need to maybe think about this in a subsequent PR.